CMakeLists.txt

```diff
@@ -6,6 +6,11 @@ include(ExternalProject)
 # Adds the cmake directory to the CMake include path.
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
+if(WIN32)
+    set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
```

Comment:
These two lines need comments, please. I have no clue what they mean.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/855/commits/64a5c981f43f9d0750eb6046a56ecae3c03277ff